tests/View/Blade/BladeComponentTagCompilerTest.php

```diff
@@ -32,30 +32,30 @@ public function testBasicComponentParsing()
 
         $result = (new ComponentTagCompiler(['alert' => TestAlertComponent::class]))->compileTags('<div><x-alert type="foo" limit="5" @click="foo" required /><x-alert /></div>');
 
-        $this->assertSame("<div> @component('Illuminate\Tests\View\Blade\TestAlertComponent', [])
+        $this->assertSame("<div> @component('Illuminate\Tests\View\Blade\TestAlertComponent', ['attributes' => ['type' => 'foo','limit' => '5','@click' => 'foo','required' => true]])
```

Comment:
Add a proper space after each array ellement:

```suggestion
        $this->assertSame("<div> @component('Illuminate\Tests\View\Blade\TestAlertComponent', ['attributes' => ['type' => 'foo', 'limit' => '5', '@click' => 'foo', 'required' => true]])
```

Please do this for all changes below.

Reply:
I've had to modify the existing function [here](https://github.com/laravel/framework/blob/7.x/src/Illuminate/View/Compilers/ComponentTagCompiler.php#L390) to make this happen. I've adjusted the tests accordingly.

Reply:
[This](https://github.com/laravel/framework/pull/32505/files#diff-6a6ea7a2d1468b4ad89232932195b939R400) is the relevant change.